models/SequenceVerificationDataCentric/models/SequenceVerificationDataCentric/SVDCdomain/AESequenceDC/SVDCclasses/ExtraJobInvariantDefn/ExtraJobInvariantDefn.masl

```diff
@@ -181,15 +169,19 @@ begin
   	              
   	       else
   	              
-             logMessage := "AESequenceDC::ExtraJobInvariantDefn.restoreInvariantOfThisDefinition : Something went wrong restoring the Extra-Job Invariant's state " & restoredInvariant.invariantName;
-             Logger::log(Logger::Debug, "AESequenceDC", logMessage);
+             if (Logger::enabled(Logger::Debug, "pv.sv.extrajobinvariantdefn.restoreInvariantOfThisDefinition")) then
+               logMessage := "AESequenceDC::ExtraJobInvariantDefn.restoreinvariantofthisdefinition : Something went wrong restoring the Extra-Job Invariant's state " & restoredInvariant.invariantName;
+               Logger::log(Logger::Debug, "pv.sv.extrajobinvariantdefn.restoreinvariantofthisdefinition", logMessage);
+             end if;
```

Comment:
Should this be Information rather than Debug?

Reply:
I agree.  You originally had it as Debug, so I kept that.  But this looks likes problems that need to be reported and acted upon.  I will promote these to Warning.